dev/com.ibm.ws.jpa.container.core/resources/com/ibm/ws/jpa/jpa.nlsprops

```diff
@@ -306,6 +306,12 @@ ERROR_TRANSFORMING_CLASS_CWWJP0055E=CWWJP0055E: An error occurred while attempti
 ERROR_TRANSFORMING_CLASS_CWWJP0055E.explanation=An unexpected error occurred while transforming the Java class. This error is usually caused by invalid Java bytecode.
 ERROR_TRANSFORMING_CLASS_CWWJP0055E.useraction=The class cache might be corrupted or the application might be packaged incorrectly. First, clear the class cache. If that action does not resolve the problem, then review the application archive for packaging mistakes.
 
+#-------------------------------------------------------------------------------
+# 0=property name, 1=property value
+EMPTY_PROP_NAME_VALUE_CWWJP0056W=CWWJP0056W: A JPA default property configured in the server.xml file contains an empty name or empty value: name = [{0}], value = [{1}]. The property will be ignored.
+EMPTY_PROP_NAME_VALUE_CWWJP0056W.explanation=Neither the name nor value attributes of a JPA default property may be empty.
```

Comment:
Suggested change to the explanation: "The name and value attributes for a JPA default property must not be empty."

Reply:
Changed as suggested.